### ISSUE TITLE
Problem with cloudfare - Waterfox 5.1.9

### ISSUE THREAD

**Issue by Felipefpl** [MrAlex94/Waterfox#3077]
### What happened?

Sites protected by cloudfare always demand to click in the checkbox to verify that i'm not a bot, this hardly happens with waterfox classic, i didnt load a lot of sites when i was going to this site.

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G5

### What platform are you seeing the problem on?

Windows

**Comment by MrAlex94** [comment]
Typically this will be due to your IP rather than the browser itself. Do you have this issue with other browsers? If not, any extensions that might be interacting with websites?

**Comment by Felipefpl** [comment]
Nope, like i said above, waterfox classic hardly have this problem, firefox is teh same.
